langchain/src/document_loaders/tests/couchbase.int.test.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from "@jest/globals";
+import { Cluster } from "couchbase";
+import { CouchbaseDocumentLoader } from "../web/couchbase.js";
+
+test("Test Couchbase Cluster connection ", async () => {
+  const connectionString = "<enter-valid-couchbase-connection-string>";
+  const databaseUsername = "<enter-valid-couchbase-user>";
+  const databasePassword = "<enter-valid-couchbase-password>";
+  const query = `
+        SELECT h.* FROM \`travel-sample\`.inventory.hotel h 
+        WHERE h.country = 'United States'
+        LIMIT 10
+    `;
+  const validPageContentFields = ["country", "name", "description"];
+  const validMetadataFields = ["id"]
+  
+  const couchbaseClient = await Cluster.connect(connectionString, {
+    username: databaseUsername,
+    password: databasePassword,
+    configProfile: "wanDevelopment"
+  });
+  const loader = new CouchbaseDocumentLoader(
+    couchbaseClient,
+    query,
+    validPageContentFields,
+    validMetadataFields
+  );
+  const docs = await loader.load();
+  console.log(docs);
```

Comment:
```suggestion
  console.log(docs);
```
I'd remove console.log

Reply:
Removed logging